Lib/fontTools/unicodedata/codepages.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+# codepage ranges mainly from:
+# https://unicode.org/Public/MAPPINGS/VENDORS/MICSFT/ 
+# https://unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WINDOWS/ 
+# https://unicode.org/Public/MAPPINGS/VENDORS/MICSFT/PC/
```

Comment:
this codepages.py module could potentially be autogenerated, see similar scripts in MetaTools

Reply:
@anthrotype Agree, but sadly not all. There is one codepage 708 I was not able to find any data I would be able to parse via script: https://github.com/moontypespace/fonttools/blob/caff9562e8bdb84e02b00c0a9d42c46c555ad292/Lib/fontTools/unicodedata/codepages.py#L39
The other character sets I have parsed via some line of code.

Reply:
how about we hard-code the odd codepage 708 only, and have a MetaTools script that generate the others that can be generted?